posthog/clickhouse/migrations/0106_materialize_session_ids_uuid.py

```diff
@@ -0,0 +1,27 @@
+from infi.clickhouse_orm import migrations
+
+from posthog.clickhouse.client.connection import get_client_from_pool
+from posthog.clickhouse.kafka_engine import trim_quotes_expr
+from posthog.settings import CLICKHOUSE_CLUSTER
+
+
+ADD_COLUMNS_SHARDED_EVENTS = f"""
+ALTER TABLE {{table}} ON CLUSTER {{cluster}}
```

Comment:
No need to use `ON CLUSTER {{cluster}}`, since `run_sql_with_exceptions` handle running the query on the needed nodes.
```suggestion
ALTER TABLE {{table}}
```